spec/book_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../book'
+
+describe Book do
+  before do
+    @book = Book.new('The Great Gatsby', 'F. Scott Fitzgerald')
+  end
+
+  describe '#initialize' do
+    it 'initializes with a title and author' do
+      expect(@book.title).to eq('The Great Gatsby')
+      expect(@book.author).to eq('F. Scott Fitzgerald')
+    end
```

Comment:
- [ ] like the previous CR mentioned, in this project you are expected to test all methods in all classes which in this class should include the `add_rental` method. you can find an example of what this might look like in the previous comment.